tests/test_yamls/test_rclone_mount.yaml

```diff
@@ -0,0 +1,24 @@
+file_mounts:
+  # Mounting private buckets in RCLONE mode
+  /mount_private_rclone:
+    name: {{storage_name}}
+    source: ~/tmp-workdir
+    store: {{store_type}}
+    mode: RCLONE
```

Comment:
Do we need this new mode, or should we directly change the behavior of `MOUNT` to rclone?

Reply:
As discussed online, we are referring this `mode` separately as `MOUNT_CACHE` mode, and keeping our original `MOUNT` mode as it is. Perhaps, open to more discussions if necessary.